tests/schema/test_lazy_types/type_a.py

```diff
@@ -20,3 +20,5 @@ def type_b(self) -> strawberry.LazyType["TypeB", ".type_b"]:  # noqa
         from .type_b import TypeB
 
         return TypeB()
+
+    done: bool = True
```

Comment:
What is this change here for? I see that it's used in `test_cyclic` but I'm wondering why. Also what drove it to be added as a resolver-less field _below_ a field with a resolver -- is that necessary?

Reply:
Due to how I evaluated forward refs it seems like the schema would not be generated in the same order, so I just had that test there to be sure.